rhel-7.template

```diff
@@ -46,6 +46,7 @@ fuse-sshfs
 nfs-utils
 go-hvkvp
 python-setuptools
+python-rhsm-certificates
```

Comment:
In RHEL this is already available due to subscription manager this is part of dependency. I just put this here just sake of having both templates same kind of packages, it shouldn't affect the RHEL iso by any mean.

```
On RHEL box
# yum deplist subscription-manager
[...]
  dependency: python-rhsm >= 1.19.10
   provider: python-rhsm.x86_64 1.19.10-1.el7_4
[...]
# yum deplist  python-rhsm 
[...]
  dependency: python-rhsm-certificates = 1.19.10-1.el7_4
   provider: python-rhsm-certificates.x86_64 1.19.10-1.el7_4
[...]

```